rkt/cli_apps.go

```diff
@@ -150,6 +155,89 @@ func (ae *appExec) Set(s string) error {
 	return nil
 }
 
+// appInjectVolume is for --inject-volume flags in the form of: --inject-volume source=PATH,target=PATH
+type appInjectVolume apps.Apps
+
+func (aam *appInjectVolume) Set(s string) error {
+	app := (*apps.Apps)(aam).Last()
+	if app == nil {
+		return fmt.Errorf("--inject-volume must follow an image")
+	}
+
+	p := strings.SplitN(s, ":", 2)
+	if len(p) != 2 {
+		return fmt.Errorf("must be SOURCE_PATH:TARGET_PATH")
+	}
+
+	volName, err := types.SanitizeACName(p[1])
+	if err != nil {
+		return fmt.Errorf("empty target in volume")
+	}
+
+	rVol := types.Volume{Name: *types.MustACName(volName), Source: p[0], Kind: "host"}
+	mount := schema.Mount{Volume: rVol.Name, Path: p[1]}
+
+	(*apps.Apps)(aam).Volumes = append((*apps.Apps)(aam).Volumes, rVol)
+	if (*apps.Apps)(aam).Count() == 0 {
```

Comment:
Is this possible to happen? We are bailing out in the beginning of this function if there is still no app...